Assets/MixedRealityToolkit/UX/Scripts/AppBar/AppBar.cs

```diff
@@ -383,54 +398,51 @@ private void RefreshTemplates ()
                     numCustomButtons++;
                 }
             }
-            List<ButtonTemplate> defaultButtonsList = new List<ButtonTemplate>();
+
+            var defaultButtonsList = new List<ButtonTemplate>();
+
             // Create our default button templates based on user preferences
             if (UseRemove)
             {
-                defaultButtonsList.Add(GetDefaultButtonTemplateFromType(ButtonTypeEnum.Remove, numCustomButtons, UseHide, UseAdjust, UseRemove));
```

Comment:
Why is `UseRemove` being removed in these lines?

Reply:
Because the parameter wasn't being used in the method, and the method was private.